### ISSUE TITLE
Ok I am stuck.

### ISSUE THREAD

**Issue by Limalex80** [pi-node/instructions#102]
Hello Guys.

I am on a Mac with BigSur 11.2.3

Docker desktop ver.: 3.2.2(61853)
Docker running on port 31400( stellar-dummy ese7en/node-port-test)

Firewall set  to allow connections for Docker and Pi Network

Port forwarding done, but Pinging does not work. the liitle wheel keep spinning

Any help would be appreciated.

<img width="604" alt="Screenshot 2021-04-08 at 11 03 35" src="https://user-images.githubusercontent.com/82149117/114008823-845b8100-985a-11eb-8b71-3bcfbca4c505.png">

**Comment by ihatejam** [comment]
The built-in port checker doesn't always work that well. 

While the "stellar-dummy" is running -- check if the ports are open using https://www.ipfingerprints.com/portscan.php 

If it also reports that your ports are closed then write down the public IP address ipfingerprints thinks you're on [Note: do NOT post your public IP address here]. 
Compare the public IP address reported by ipfingerprints with the public IP address reported by the WAN interface on your router (you'll need to log into your router config). If the two IP addresses do not match, then it is likely that your ISP does not hand out "real" public IP addresses. This is common practice in places where the demand for IP addresses is larger than the IP address block allocated to your ISP. To resolve this, you can do the following:
a) ask your ISP if they can give you an IP address that allows port forwarding. 
b) switch to an ISP that allows/facilitates port forwarding.
c) use a VPN provider that automatically takes care of port forwarding like [Surfshark](https://surfshark.club/friend/b5BmtEtn?locale=en)

**Comment by mokhabarat** [comment]
hello
you're bot running the docker image responsible for actually running pi
sometimes it gets stuck. open docker and run it manually and you should be up and running. also get a vpn much easier to open ports

**Comment by Limalex80** [comment]
Thanks Guys. Look like the problem was the ISP. So I went for the VPN service. ip fingerprints now shows all port open and docker reports server listening on all ports, but pinging still not working.
<img width="602" alt="Screenshot 2021-04-09 at 03 03 12" src="https://user-images.githubusercontent.com/82149117/114118339-8ec05e00-98e0-11eb-9858-5d2819f634fc.png">

After this picture I re checked and now not reporting any ports , but only a red cross apiers

**Comment by Limalex80** [comment]
After restart
<img width="610" alt="Screenshot 2021-04-09 at 03 13 33" src="https://user-images.githubusercontent.com/82149117/114118999-b4019c00-98e1-11eb-9991-aa78671a9d37.png">

**Comment by Limalex80** [comment]
Every time I checking port settings changing

**Comment by nicolae788** [comment]
Just hit continue and let the thing do the job. You don't have to get everything perfect :))). Anyway the consensus will be downloaded after days of waiting and then you will see that it uses 1 port of the 10.

**Comment by ihatejam** [comment]
> Every time I checking port settings changing

You're good to go if ipfingerprints reports them as open -- just continue on and then wait for the system to select you as a candidate to run the consensus container - this part takes a while. When you're selected, the consensus container will be pushed down to your setup --> then the stellar-dummy will be stopped -->and the consensus container will be started up which will in turn interact with other live testnet blockchain nodes.

**Comment by Limalex80** [comment]
Lovely thank you guys

**Comment by lanctot6** [comment]
Great Info here!!
When I run https://www.ipfingerprints.com/portscan.php , I get that the ports are filtered. Anything to look at specifically with this info? 
![Screen Shot 2021-04-24 at 3 20 44 PM](https://user-images.githubusercontent.com/21003062/115973231-a6b6f500-a510-11eb-927a-5c16fcc7ad1f.png)

**Comment by lanctot6** [comment]
Oh SICK!! used that safeshark vpn and all is Good.
![Screen Shot 2021-04-24 at 3 30 31 PM](https://user-images.githubusercontent.com/21003062/115973418-0366df80-a512-11eb-83df-f6fb5b110b1f.png)

Do we just chill and wait now?

**Comment by ihatejam** [comment]
Yep - pretty much just have to chill and wait. Leave it running so they can monitor your uptime. Eventually, you'll be selected to run the consensus container.

**Comment by nathusius** [comment]
Is it possible to reset the availability % figure somehow?! I thought I had a problem with the port access when actually it was all OK, so now I am leaving it all running 24 hours a day, but still only have 55% availability since installation on 1st April and that figure is only increasing very slowly of course! Or will the system spot that I now have more or less permanent availability, even though overall % value doesn't reflect this?

**Comment by ihatejam** [comment]
> Is it possible to reset the availability % figure somehow?! I thought I had a problem with the port access when actually it was all OK, so now I am leaving it all running 24 hours a day, but still only have 55% availability since installation on 1st April and that figure is only increasing very slowly of course! Or will the system spot that I now have more or less permanent availability, even though overall % value doesn't reflect this?

You can't reset your availability - Don't worry though - you'll still be picked up as a node.

**Comment by nathusius** [comment]
OK, thought as much, thanks for confirming!
